bundle/manifests/shipwright-operator.clusterserviceversion.yaml

```diff
@@ -212,88 +212,168 @@ spec:
       clusterPermissions:
       - rules:
         - apiGroups:
-          - ""
+          - shipwright.io
           resources:
-          - pods
-          - services
-          - services/finalizers
-          - endpoints
-          - persistentvolumeclaims
-          - events
-          - configmaps
-          - secrets
-          - serviceaccounts
+          - buildruns
           verbs:
-          - create
+          - get
+          - list
+          - watch
+          - update
           - delete
+        - apiGroups:
+          - shipwright.io
+          resources:
+          - buildruns/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - shipwright.io
+          resources:
+          - buildruns/status
+          verbs:
+          - update
```

Comment:
This could be merged into a single entry.

Reply:
Everything under `/bundle` is auto-generated by operator-sdk, and will be used in the listing on operatorhub.io.